deploy/controller/deployment.yaml

```diff
@@ -14,18 +14,38 @@ spec:
       labels:
         name: provider-credential-controller
     spec:
+      tolerations:
+      - key: dedicated
+        operator: Exists
+        effect: NoSchedule
+      - effect: NoSchedule 
+        key: node-role.kubernetes.io/infra 
+        operator: Exists
+      serviceAccountName: provider-credential-controller
+      hostNetwork: false
+      hostPID: false
+      hostIPC: false
+      securityContext:
+        runAsNonRoot: true
+
       containers:
       - command:
         - "./manager"
         - "-enable-leader-election"
-        image: quay.io/jpacker/provider-credential-controller:0.1
+        image: registry.ci.openshift.org/open-cluster-management/2.3:provider-credential-controller
```

Comment:
not pushing this image to quay?

Reply:
It gets pushed to quay.io via ff on 2.3, but for the opensource reference, I used the PROW image, not the quay image.

I don't think we have any guidance on which they want to use either.  The install which leverages the cluster-lifecycle-chart will use a quay.io value.